### PR TITLE
HPCC-15117 Document Virtual Slaves

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@jakesmith Please review

### CODE REVIEW

File: docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml [hunk 1]
```diff
@@ -1547,6 +1547,175 @@ lock=/var/lock/HPCCSystems</programlisting>
...
+            <para>Each slave process in this configuration has an equal fixed
+            split of the physical memory available to the node.</para>
+          </listitem>
+
+          <listitem>
+            <para>Slaves do not share RAM or any other resources.</para>
+          </listitem>
+
+          <listitem>
+            <para>Communication between slaves uses message passing and
```
Comment (jakesmith) [inline docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml]: uses -> use
and loopback device -> via the lookpback network interface

**Comment by jakesmith** [comment]
@JamesDeFabia - one minor comment.

**Comment by JamesDeFabia** [comment]
@jakesmith Edits made, please have another look

**Comment by jakesmith** [comment]
Looks good to merge.
@richardkchapman

**Pull request merged**